cmd/play/main.go

```diff
@@ -77,22 +78,25 @@ func genCmd() *cobra.Command {
 				config.Namespace = "refunc-play"
 			}
 			if err := k8sTpl.Execute(os.Stdout, struct {
-				Namespace string
-				ImageTag  string
-				Bucket    string
-				S3Prefix  string
+				Namespace 		string
+				ImageTag  		string
+				Bucket    		string
+				S3Prefix  		string
+				UploadSizeLimit string
 			}{
 				Namespace: config.Namespace,
-				ImageTag:  version.Version,
-				Bucket:    tplConfig.Bucket,
-				S3Prefix:  tplConfig.S3Prefix,
+				ImageTag:  		 version.Version,
+				Bucket:    		 tplConfig.Bucket,
+				S3Prefix:  		 tplConfig.S3Prefix,
+				UploadSizeLimit: tplConfig.UploadSizeLimit,
 			}); err != nil {
 				klog.Fatal(err)
 			}
 		},
 	}
 	cmd.Flags().StringVar(&tplConfig.Bucket, "bucket", "refunc", "Bucket for minio to store functions")
 	cmd.Flags().StringVar(&tplConfig.S3Prefix, "prefix", "functions", "Path prefix(folder) to store functions under bukect")
+	cmd.Flags().StringVar(&tplConfig.UploadSizeLimit, "uploadSizeLimit", "functions", "Max upload size (in megabyte) for lambda archive")
```

Comment:
This flags third parameter is default value. This `functions` string is not work for nginx's `client_max_body` setting.